queue/src/fiskaltrust.Middleware.Localization.QueueDEFAULT/Extensions/ChargeItemExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using fiskaltrust.ifPOS.v1;
+using fiskaltrust.Middleware.Contracts.Exceptions;
+
+namespace fiskaltrust.Middleware.Localization.QueueDEFAULT.Extensions
+{
+    public static class ChargeItemExtensions
```

Comment:
We should also add an explanation comment which explains that these extensions should be used to get information from the `ChargeItem` like which `ftChargeItemCase` or `ftChargeItemCaseFlag`s are set. Maybe also link to Italy as an example.

(same for the other Extensions 🙏)